source/clear-linux/tutorials/aws-web/aws-web.rst

```diff
@@ -0,0 +1,266 @@
+      :guilabel:`|CL| OS Basic` AMI along with the pricing details for running
+      |CL| on different platform configurations as shown in figure 4. Select
+      the :guilabel:`Continue` button.
+
+      .. figure:: figures/aws-web-4.png
+         :scale: 50 %
+         :alt: Clear Linux OS Basic
+
+         Figure 4: :guilabel:`Clear Linux OS Basic`
+
+#. The :guilabel:`Choose Instance Type` menu appears as shown in figure 5.
+
+   Select the :guilabel:`t2.micro` type by clicking the box on the left side
+   of the instance and then select the :guilabel:`Review and Launch` button to
+   move to the :guilabel:`Step 7: Review the Instance Launch` menu.
+
+   .. note::
+
+      You can configure the instance details, add additional storage, add
+      tags, and configure the security group before selecting the
+      :guilabel:`Review and Launch` button if you want to further customize
+      this |CL| instance.
+
+   .. figure:: figures/aws-web-5.png
+      :scale: 50 %
+      :alt: Choose an Instance Type
+
+      Figure 5: :guilabel:`Choose an Instance Type`
+
+#. The :guilabel:`Step 7: Review the Instance Launch` menu allows you to
+   :guilabel:`Cancel` the process, return to the :guilabel:`Previous` screen
+   to change the configuration or :guilabel:`Launch` the instance defined.
+
+   .. figure:: figures/aws-web-6.png
+      :scale: 50 %
+      :alt: Step 7: Review the Instance Launch
+
+      Figure 6: :guilabel:`Step 7: Review the Instance Launch`
+
+   #. Select the :guilabel:`Launch` button. A dialog box appears, as shown in
+      figure 7, asking you to
+      :guilabel:`Select an existing key pair or create a new pair`.
+
+      .. figure:: figures/aws-web-7.png
+         :scale: 50 %
+         :alt: Select an existing key pair or create a new pair
+
+         figure 7: :guilabel:`Select an existing key pair or create a new pair`
+
+      #. Select the :guilabel:`Create a new key pair` option.
+      #. For the :guilabel:`Key pair name`, enter `AWSClearTestKey`.
+      #. Select the :guilabel:`Download Key Pair` button to download the
+         :file:`AWSClearTestKey.pem` to your browser's defined
+         :file:`Downloads` directory.
+      #. When the file finishes downloading, select the
+         :guilabel:`Launch Instances` button to proceed to the
+         :guilabel:`Launch Status` menu shown in figure 8.
+
+         .. figure:: figures/aws-web-8.png
+            :scale: 50 %
+            :alt: Launch Status
+
+            figure 8: :guilabel:`Launch Status`
+
+      #. Once the :guilabel:`Launch Status` page changes to what is shown in
+         figure 9, select the :guilabel:`View Instances` button to view your
+         :guilabel:`Instances` dashboard.
+
+         .. figure:: figures/aws-web-9.png
+            :scale: 50 %
+            :alt: View Instance
+
+            figure 9: :guilabel:`View Instance`
+
+Connect to your Clear Linux OS Basic instance
```

Comment:
basic

Reply:
`Clear Linux OS Basic` is the name of the AMI